.github/workflows/code-hygiene.yml

```
@@ -0,0 +1,11 @@
+name: Code Hygiene
+on: [pull_request]
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)